lib/yoti/data_type/profile.rb

```diff
@@ -1,77 +1,69 @@
 module Yoti
   # Encapsulates Yoti user profile
   class Profile
-
     def initialize(profile_data)
-        if !profile_data.is_a? Object
-            profile_data = Hash.new
-        end
-        @profile_data = profile_data
+      profile_data = {} unless profile_data.is_a? Object
+      @profile_data = profile_data
     end
 
     def selfie
-        return get_attribute(Yoti::Attribute::SELFIE)
+      get_attribute(Yoti::Attribute::SELFIE)
     end
 
     def family_name
-        return get_attribute(Yoti::Attribute::FAMILY_NAME)
+      get_attribute(Yoti::Attribute::FAMILY_NAME)
     end
 
     def given_names
-        return get_attribute(Yoti::Attribute::GIVEN_NAMES)
+      get_attribute(Yoti::Attribute::GIVEN_NAMES)
     end
 
     def full_name
-        return get_attribute(Yoti::Attribute::FULL_NAME)
+      get_attribute(Yoti::Attribute::FULL_NAME)
     end
 
     def phone_number
-        return get_attribute(Yoti::Attribute::PHONE_NUMBER)
+      get_attribute(Yoti::Attribute::PHONE_NUMBER)
     end
 
     def email_address
-        return get_attribute(Yoti::Attribute::EMAIL_ADDRESS)
+      get_attribute(Yoti::Attribute::EMAIL_ADDRESS)
     end
 
     def date_of_birth
-        return get_attribute(Yoti::Attribute::DATE_OF_BIRTH)
+      get_attribute(Yoti::Attribute::DATE_OF_BIRTH)
     end
 
     def gender
-        return get_attribute(Yoti::Attribute::GENDER)
+      get_attribute(Yoti::Attribute::GENDER)
     end
 
     def nationality
-        return get_attribute(Yoti::Attribute::NATIONALITY)
+      get_attribute(Yoti::Attribute::NATIONALITY)
     end
 
     def postal_address
-        postal_address = get_attribute(Yoti::Attribute::POSTAL_ADDRESS)
+      postal_address = get_attribute(Yoti::Attribute::POSTAL_ADDRESS)
+      return postal_address unless postal_address.nil?
 
-        return postal_address unless postal_address.nil?
-        return get_formatted_address
+      formatted_address
     end
 
     def structured_postal_address
-        return get_attribute(Yoti::Attribute::STRUCTURED_POSTAL_ADDRESS)
+      get_attribute(Yoti::Attribute::STRUCTURED_POSTAL_ADDRESS)
     end
 
     # @return attribute value by name
     def get_attribute(attr_name)
-        if @profile_data.has_key? attr_name then
-            return @profile_data[attr_name]
-        end
-        return nil
+      @profile_data[attr_name] if @profile_data.key? attr_name
```

Comment:
I think the old way of doing this is clearer! Could you share the complaint that RuboCop has when it is that way?

Reply:
I'll think I can make this clearer keeping in line with the coding style - here are the errors that were addressed:

```
lib/yoti/data_type/profile.rb:58:7: C: Style/IfUnlessModifier: Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||. (https://github.com/rubocop-hq/ruby-style-guide#if-as-a-modifier)
      if @profile_data.key? attr_name then
      ^^
```

```
lib/yoti/data_type/profile.rb:58:39: C: Style/MultilineIfThen: Do not use then for multi-line if. (https://github.com/rubocop-hq/ruby-style-guide#no-then)
      if @profile_data.key? attr_name then
                                      ^^^^
```

```
lib/yoti/data_type/profile.rb:62:7: C: Style/RedundantReturn: Redundant return detected. (https://github.com/rubocop-hq/ruby-style-guide#no-explicit-return)
      return nil
      ^^^^^^
```

Reply:
I've changed this to:
```ruby
  def get_attribute(attr_name)
    return nil unless @profile_data.key? attr_name

    @profile_data[attr_name]
  end
```